actions/tsconfig.json

```diff
@@ -9,6 +9,6 @@
     "target": "es2020",
     "resolveJsonModule": true
   },
-  "exclude": ["**/*.spec.ts"],
+  // "exclude": ["**/*.spec.ts"],
```

Comment:
Why commenting it out rather than removing?

Reply:
Its good to add I believe, just avoid compiling tests which is not needed

https://stackoverflow.com/questions/36475736/exclude-with-pattern-in-tsconfig-json

Reply:
I may not understand the `jest` foo here, but from my understanding, `jest` doesn't work with typescript, and as the tests are written in typescript, these need to be compiled (otherwise, there would be the need for babel or something to transpile? I left this with the simplest I knew which is just compile everything, including the tests, and then run `jest` on this.